release_notes/44-release-notes.adoc

```diff
@@ -645,6 +646,33 @@ The following changes are included:
 
 * This release updates the process to ensure that Sensor completes the TLS handshake during secret synchronization. If this does not happen, the process can freeze and leave Sensor with incomplete data stores and a corrupted state, even if Sensor appears healthy.
 
+[id="resolved-in-version-445_{context}"]
+=== Resolved in version 4.4.5
+
+*Release date*: 29 August 2024
+
+This release of {product-title-short} fixes the following bugs:
+
+* Fixed an issue where Sensor could not identify pull secrets because the annotation was changed from `kubernetes.io service-account.name` to `openshift.io/internal-registry-auth-token.service-account` in the {ocp}.
+
+* Fixed an issue where policies incorrectly displayed an enforced value of `Yes` in the *Configuration Management -> Application & Infrastructure -> Deployments* page due to inconsistent status handling, while showing `No` in the *Violations* page.
+
+* Fixed an issue where outdated `nghttp2` libraries in various container images had vulnerabilities due to old base images.
+
+* Fixed an issue where outdated `libuv` versions in various container images had vulnerabilities due to the use of old base images.
+
+* Fixed an issue where outdated `python-idna` libraries in various container images had vulnerabilities due to old base images.
+
+* Fixed an issue where vulnerability data differed between Vulnerability Management 2.0 and Vulnerability Management 1.0 by using ScannerV4 due to inconsistencies in vulnerability reporting, affecting images such as `quay.io/openshift-release-dev/ocp-v4.0-art-dev`.
```

Comment:
ScannerV4 -> Scanner V4. Though, I don't think this is limited to just Scanner V4. I think you can remove the reference to the particular scanner in this note